File: Week 1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Ask for the Height
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    // Each row
    for (int i = 0; i < height; i++)
    {

        // Each column print spaces
        for (int k = (height - i); k > 1; k--)
        {
            printf(" ");
        }
        // Each column print #
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        // Move to next line
        printf("\n");
    }
}

File: Week 6 - Python/sentimental-mario-more/mario.py
# Program that prints out a half-pyramid of a specified height

def main():
    # Loop for input if outside range 1-8
    while True:
        try:
            get_int = int(input("Height: "))
            if get_int >= 1 and get_int <= 8:
                break
            else:
                print("Enter a number between 1-8")
        except ValueError:
            print("Enter a number between 1-8")

    # Loop through height input, for each line, print space and pound symbols
    for row in range(get_int):
        spaceCount = get_int - (row+1)
        poundCount = row+1
        printSpace(spaceCount)
        printPound(poundCount)
        print("  ", end="")
        printPound(poundCount)
        print()

# Function to print spaces


def printSpace(spaces):
    for i in range(spaces):
        print(" ", end="")
    return

# Function to print pound symbol


def printPound(pound):
    for i in range(pound):
        print("#", end="")
    return


if __name__ == "__main__":
    main()
